internal/talkerpb/talkerpb.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Lumberjack authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate protoc -I../../third_party/googleapis -I../../internal/protos/talker --go_out=. --go-grpc_out=. --go_opt=paths=source_relative --go-grpc_opt=paths=source_relative talker.proto
+
+package talkerpb
```

Comment:
It probably doesn't have any practical impact but it'd be good if we update https://github.com/abcxyz/lumberjack/blob/7b83e6469d094af974acedb1209ae7377676214f/internal/protos/talker/talker.proto#L18 to be `option go_package = "github.com/abcxyz/lumberjack/internal/talkerpb";`